.cirrus.yml

```diff
@@ -308,7 +308,7 @@ alt_build_task:
 
 # Confirm building the remote client, natively on a Mac OS-X VM.
 osx_alt_build_task:
-    name: "OSX Cross"
+    name: "Build for OSX darwin/arm64"
```

Comment:
We're building for both arm and amd, I could see this as possibly confusing that we're only building for arm, maybe `Build for OSX on darwin/arm64` Or just Build for OSX?

Reply:
Mostly a nit though

Reply:
If we are niting the name might as well rename OSX to macOS, OSX is no used as the recent versions no longer include 10 in the version. It is just macOS 14 Sonoma now.

Reply:
Great feedback, thanks guys.